src/evm_transition_tool/types.py

```diff
@@ -98,4 +98,4 @@ class TransitionToolOutput(CamelModel):
 
     alloc: Alloc
     result: Result
-    body: Bytes
+    body: Bytes | None = None
```

Comment:
What is body?

Reply:
According to geth's source code, it's the rlp encoded list of transactions, but EEST does it internally so the result is discarded.